### ISSUE TITLE
init script Clarification

### ISSUE THREAD

**Issue by adminy** [tinycorelinux/Core-scripts#37]
My understanding is that initrd is a gzipped cpio that the kernel loads into ram.
However here:
https://github.com/tinycorelinux/Core-scripts/blob/f9f48e9fb6929e30fdbb12aff0f1efbe66b363f1/init#L6-L16

 I see that tinycore redoes this operation again, by copying itself into ram, is there any reason why that's done? I don't understand and I couldn't find why this was done in the book. The only hint I can take is the 90%  size but I couldn't find reference in the man to that either.

Thank you,
Marin

**Comment by clbr** [comment]
That is the "noembed" path. If you google for that, you should find some hits. It's a non-default path for some third-party software that expects to find / in various mounted lists, for their installers to work, etc.

**Comment by adminy** [comment]
I found this in the core book:

> 10.32. noembed - use a separate tmpfs
This is an advanced option that changes where in RAM Core is run
from. By default, Core uses the tmpfs setup by the kernel; with this
bootcode, Core will setup a new tmpfs file system, and use that
instead.
Using this bootcode temporarily doubles the RAM use, as both
copies are kept in RAM at once during boot. As an extra copy is
made, it also slows the boot time. It allows GNU df to detect the
free space in /, used by some proprietary software installers.

And if I look at the Pi branch of this repo, I find that noembed is set as a cmdline arg.
But that's why it makes no since since something like a pi has very little memory.

I tried running `df` and the `/` does show up anyways, so I'm really not sure why this is really needed.

Also I still don't understand what the 90% does, and why we need the `nr_inodes` arg
